test/json-ld.net.tests/ConformanceTests.cs

```diff
@@ -77,20 +74,21 @@ public ConformanceCases()
 
         public IEnumerator<object[]> GetEnumerator()
         {
+            var jsonFetcher = new JsonFetcher();
```

Comment:
I created the `JsonFetcher` class so I could reuse some of this logic in my own test class